pkg/querier/tripperware/roundtrip.go

```diff
@@ -113,6 +111,11 @@ func NewQueryTripperware(
 		Help: "Total queries sent per tenant.",
 	}, []string{"op", "user"})
 
+	rejectedQueriesPerTenant := promauto.With(registerer).NewCounterVec(prometheus.CounterOpts{
+		Name: "cortex_query_frontend_rejected_queries_total",
+		Help: "Total rejected queries per tenant.",
+	}, []string{"user"})
```

Comment:
Does it make sense to add another label to distinguish different APIs being throttled?

Reply:
done